akka-docs/src/main/paradox/java/stream/stages-overview.md

```diff
@@ -319,8 +320,8 @@ after this the stream is canceled. If no element is emitted, the CompletionStage
 
 ### headOption
 
-Materializes into a `CompletionStage<Optional<T>>` which completes with the first value arriving wrapped in optional,
-or an empty optional if the stream completes without any elements emitted.
+Materializes into a @scala[`Future[Option[T]]`] @java[`CompletionStage<Optional<T>>`] which completes with the first value arriving wrapped in @scala[`Some`] @java[`optional`],
```

Comment:
In Java the materialized value is wrapped in java.util.Optional.. hence it's Optional in java docs